README.md

```diff
@@ -46,6 +44,33 @@
 
 ![Deploy](resources/Deploy.gif)
 
+## Language-Specific Prerequisites
+
+### JavaScript
+
+* [Node 8.0+](https://nodejs.org/)
+
+### C#
+
+* [VS Code Debugger for C#](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp)
+* [Dotnet cli](https://docs.microsoft.com/dotnet/core/tools/?tabs=netcore2x)
```

Comment:
Is ".NET Core CLI" the official name?

Reply:
I changed it to '.NET CLI'. I'm not sure if 'Core' is supposed to be in there because you can use 'dotnet new' for '.NET Framework' templates